src/components/GraphiQL.js

```diff
@@ -382,13 +382,14 @@ export class GraphiQL extends React.Component {
                 editorTheme={this.props.editorTheme}
                 ResultsTooltip={this.props.ResultsTooltip}
               />
-              <GraphiQL.Footer>
-                {this.props.plugins
-                  ? <PluginsPane
+              {this.props.plugins
+                ? <GraphiQL.Footer>
```

Comment:
So if there's no plugins we don't render the plugin pane?

Reply:
Yup, we just render the original footer like they do here: https://github.com/graphql/graphiql/blob/master/src/components/GraphiQL.js#L377